code/src/public/index.html

```diff
@@ -32,20 +34,21 @@ <h1 class="header__title">
             <nav class="header__nav" role="navigation" aria-label="Main navigation">
                 <ul class="nav-list">
                     <li class="nav-list__item">
-                        <a href="/api/students" 
-                           class="nav-list__link" 
-                           hx-get="/api/students" 
-                           hx-target="#main-content"
-                           hx-push-url="true">
+                        <a href="/api/classes/my-classes?userId=cmhshve9l0005h5t98gm3ek8w" class="nav-list__link"
```

Comment:
It looks like the userId is hardcoded. Should this ideally come from the session or request context? Since it might be outside your current scope, it’s fine to leave the page untied for now.